packages/plugins/sentry/src/index.ts

```diff
@@ -191,6 +191,14 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
           op,
           tags,
         });
+
+        if (!rootSpan) {
+          const error = [
+            `Could not create the root Sentry transaction for the GraphQL operation "${transactionName}".`,
+            `It's very likely that this is because you have not included the Sentry tracing SDK in your app's runtime before handling the request.`,
```

Comment:
@orta Do you think it would be nice to also include a link here that points to the corresponding sentry documentation?

Reply:
We can do this in a follow-up PR.